app/controllers/subscriptions_controller.rb

```diff
@@ -1,5 +1,7 @@
 class SubscriptionsController < ApplicationController
   def create
+    return render json: {}, status: status if notify_address
```

Comment:
At this point status will be undefined (ruby probably hoists it to be a nil) probably be better if this was set to `:created`.

The json returned should also return the same response as the rest of the app as this would break any apps expecting an id in the response. I'd go with an id of 0.